### PR TITLE
Graph Colours and Shapes #164

### PR DESCRIPTION
**Pull request by lposo**
Now, users can specify the colour and shape for a given dataset on the graph. There are four choices for shape. Colours can be changed by using the colour picker.

image

(edit): Further changes have been made:
image

### CODE REVIEW

**Review by ds161833 — CHANGES_REQUESTED** [review]
image
can you have it say "circle" by default
also instad of displayed a text, can you just display a little colored squeare maybe
image
so that people can identify their datasetsby the color, by text it's pretty hard
Other than that, It's very smooth, I like it, good job!

**Review by ds161833 — CHANGES_REQUESTED** [review]
image
can you also have the two buttons be rightmost on the dataset row? so just swap color/shape picker, with the eye and bin buttons

**Review by ds161833 — APPROVED** [review]

**Review by ds161833 — APPROVED** [review]
nice! why did you remove package-lock.json though
image

File: Client/src/Models/Datasets/IDatasetRowModel.tsx [hunk 1]
```diff
@@ -10,6 +10,8 @@ export const newDatasetRow: IDatasetRowModel = {
   id: null,
   name: null,
   isInitiallyHidden: false,
-  color: null,
+  color: "circle",
```
Comment (ds161833) [inline Client/src/Models/Datasets/IDatasetRowModel.tsx]: wait, color circle?

Comment (ds161833) [reply Client/src/Models/Datasets/IDatasetRowModel.tsx]: you meant to have "shape" circle?

Comment (ds161833) [reply Client/src/Models/Datasets/IDatasetRowModel.tsx]: add a default color too actually

**Review by ds161833 — CHANGES_REQUESTED** [review]
image
override the default bullet.stroke too

File: Client/src/Models/Datasets/IDatasetRowModel.tsx (hunk 1 above)
Comment (lposo) [reply Client/src/Models/Datasets/IDatasetRowModel.tsx]: Fixed in latest push

**Review by ds161833 — APPROVED** [review]

**Review by laithAwad — APPROVED** [review]

**Pull request merged**